app/javascript/components/visualization/ScatterPlot.jsx

```diff
@@ -186,6 +188,7 @@ function RawScatterPlot({
         }
       })
     }
+    scatter.hasArrayLabels = scatter.data.annotations.some(annot => annot.includes('|'))
```

Comment:
I'm curious how often vertical pipes -- i.e. | -- occur in pre-existing group labels / annotation labels.  I imagine it's infrequent, but in cases where it doesn't denote an array this would be a bug.

Reply:
I can't run the analysis for cluster-specific annotations, but all study-wide annotations in production that have `|` characters in labels are clearly intended as arrays